client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -243,8 +243,11 @@ class DraftailInsertBlockCommand {
 
     this.blockMax = addSibling.getBlockMax(blockDef.name);
     this.icon = blockDef.meta.icon;
-    this.description = blockDef.meta.label;
+    this.label = blockDef.meta.label;
```

Comment:
This and in `render` below – I’m not entirely sure why I chose `description` rather than `label` before. Either way this needs changing to avoid the clash with the `description` now used for the ComboBoxPreview.